src/main/java/ro/jlg/staff/manager/application/DepartmentService.java

```diff
@@ -0,0 +1,56 @@
+package ro.jlg.staff.manager.application;
+
+import org.springframework.stereotype.Service;
+import ro.jlg.staff.manager.application.dto.AddDepartmentDTO;
+import ro.jlg.staff.manager.application.dto.ModifyDepartmentDTO;
+import ro.jlg.staff.manager.domain.Department;
+import ro.jlg.staff.manager.domain.DepartmentRepository;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class DepartmentService {
+
+    private DepartmentRepository departmentRepository;
+
+    // constructor
```

Comment:
Our code should express it's intent without having to use commentaries. In general, it's a bad practice.